src/utils/change-case.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Converts a string to camelCase format
+ *
+ * @param {string} str String to be converted
+ * @returns converted string
+ */
+export const toCamelCase = (str: string) => {
+  return str.replace(/([-_\s]+[a-z])/gi, $1 => {
+    return $1
+      .toUpperCase()
+      .replace(/-/g, '')
+      .replace(/_/g, '')
+      .replace(/\s/g, '');
+  });
+};
+
+/**
+ * Converts a string to camelCase format
+ *
+ * @param {string} str String to be converted
+ * @returns converted string
+ *
+ * @see {@link https://regex101.com/r/EMrqS7/1}
+ */
+export const toCssVariable = (str: string) => {
+  return `-${str.replace(/([_\s]+\w|(?<=[a-z])[A-Z\d]|^\w)/g, $1 => {
+    return `-${$1.toLowerCase()}`;
```

Comment:
This doesn't strip out spaces or underscores. So `hello_how are_you` becomes `--hello-_how- are-_you`. As above, maybe `-{$1.substr(-1).toLowerCase()}` would work ?

Reply:
Should be `return `-${$1.substr(-1).toLowerCase()}`;`, but yes...

Reply:
(assuming this was probably actually a git hub stripping out characters issue with your comment!)

Reply:
Oops, yes I clearly forgot to do replacement. I'm adding some unit tests for good measure (and to `toCamelCase()`) to verify I've covered all bases.

Reply:
Removed in favour of the tried and tested lodash